cypress-tests/cypress/e2e/v2/projectResources.cy.ts

```diff
@@ -0,0 +1,214 @@
+    cy.getDataCy("data-connector-mount-input").should("have.value", name);
+    cy.getDataCy("data-connector-edit-update-button").click();
+    cy.getDataCy("data-connector-edit-success").should("be.visible");
+    cy.getDataCy("data-connector-edit-close-button").click();
+
+    // ? Currently, data connectors newly linked might not appear immediately
+    visitCurrentProject();
+    cy.getDataCy("data-connector-box")
+      .find(`[data-cy=data-connector-name]`)
+      .contains(name);
+
+    // Edit data connector
+    const newName = `${name} edited`;
+
+    cy.getDataCy("data-connector-box")
+      .find(`[data-cy=data-connector-name]`)
+      .contains(name)
+      .click();
+    cy.getDataCy("data-connector-edit").click();
+    cy.getDataCy("test-data-connector-button").click();
+    cy.getDataCy("add-data-connector-continue-button").click();
+
+    cy.getDataCy("data-connector-name-input")
+      .should("have.value", name)
+      .clear()
+      .type(newName);
+    cy.getDataCy("data-connector-slug-input").should("have.value", name);
+    cy.getDataCy("data-connector-mount-input").should("have.value", name);
+    cy.getDataCy("data-connector-edit-update-button").click();
+    cy.getDataCy("data-connector-edit-success").should("be.visible");
+    cy.getDataCy("data-connector-edit-close-button").click();
+
+    // ? Currently, data connectors newly linked might not appear immediately
+    visitCurrentProject();
+    cy.getDataCy("data-connector-box")
+      .find(`[data-cy=data-connector-name]`)
+      .contains(newName);
+  });
+
+  it("Work with code", () => {
+    const repoUrl = "https://github.com/SwissDataScienceCenter/renku-ui.git";
+    const repoName = "renku-ui";
+    const repoUrlEdited =
+      "https://github.com/SwissDataScienceCenter/renku-data-services.git";
+    const repoNameEdited = "renku-data-services";
+
+    // Add code repository
+    visitCurrentProject();
+    cy.getDataCy("add-code-repository").click();
+    cy.getDataCy("project-add-repository-url").should("be.empty").type(repoUrl);
+    cy.getDataCy("add-code-repository-modal-button").click();
+    cy.getDataCy("code-repositories-box")
+      .find("[data-cy=code-repository-item]")
+      .contains(repoName);
+    cy.contains("[data-cy=code-repository-item]", repoName).contains(
+      "Pull only",
+    );
+
+    // Edit code repository
+    cy.contains("[data-cy=code-repository-item]", repoName)
+      .find("[data-cy=code-repository-edit]")
+      .click();
+    cy.getDataCy("project-edit-repository-url")
+      .should("have.value", repoUrl)
+      .clear()
+      .type(repoUrlEdited);
+    cy.getDataCy("edit-code-repository-modal-button").click();
+    cy.getDataCy("code-repositories-box")
+      .find("[data-cy=code-repository-item]")
+      .contains(repoNameEdited);
+    cy.contains("[data-cy=code-repository-item]", repoNameEdited).contains(
+      "Pull only",
+    );
+  });
+
+  it("Work with session environments", () => {
```

Comment:
This makes it sound like you fully tested the environment - i.e. like we launched a session. But that is not the case.

```suggestion
  it("Add and modify session environments", () => {
```